explain.py

```diff
@@ -159,6 +160,14 @@ def _explain_hash(self) -> Tuple[str, Dict[str, str]]:
                             "Buckets are doubled until there are enough, so they are always a power of 2."
         }, **self._generic_explain_dict())
 
+    def _explain_sorted_merge(self) -> Tuple[str, Dict[str, str]]:
+       return f"A sorted merge join is performed on {self.hash_cond}.", dict({
+            "Description": "Sorted Merge Join is when two lists are sorted on their join keys before being joined together. "
+                           "Postgres then traverse over the two lists in order, finding pairs that have identical join keys"
+                           " and returning them as a new, joined row.\n",
+            "Join type": self.join_type
+        }, **self._generic_explain_dict()) 
+
```

Comment:
See above comment, you will need 2 explain methods now, one for sort one for merge. Try to see the raw json and try to customise the dictiionary as detailed as possible. You can get inspiration from the plan in the link i shared which visualizes a sort merge query.